sys-fs/cryfs/cryfs-9999.ebuild

```diff
@@ -40,6 +40,9 @@ src_prepare() {
 	# remove tests that require internet access to comply with Gentoo policy
 	sed -i -e '/CurlHttpClientTest.cpp/d' -e '/FakeHttpClientTest.cpp/d' test/cpp-utils/CMakeLists.txt || die
 
+	# remove non-applicable warning caused by cmake-utils setting NDEBUG
+	sed -i -e '/WARNING! This is a debug build. Performance might be slow./d' src/cryfs-cli/Cli.cpp || die
```

Comment:
Yes, I suppose that's the case. However, if they want to account for specific CMAKE_BUILD_TYPE, they should really verify CMAKE_BUILD_TYPE rather than the side effects of it.